installing/disconnected_install/about-installing-oc-mirror-v2.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oc-mirror-v2"]
+= Mirroring images for a disconnected installation using the V2
+include::_attributes/common-attributes.adoc[]
+:context: installing-mirroring-disconnected
+
+toc::[]
+
+:FeatureName: oc-mirror v2
+include::snippets/technology-preview.adoc[]
```

Comment:
In the preview, this is "v2" not "V2." I would use whichever you want throughout.

Reply:
Command line text should be verbatim; for text, I would just choose whichever the PM wants customers to see.